packages/commonwealth/client/scripts/views/modals/TradeTokenModel/TradeTokenForm/AmountSelections/SellAmountSelection.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { CWIcon } from 'views/components/component_kit/cw_icons/cw_icon';
+import { CWText } from 'views/components/component_kit/cw_text';
+import { CWTag } from 'views/components/component_kit/new_designs/CWTag';
+import { CWTextInput } from 'views/components/component_kit/new_designs/CWTextInput';
+import { SellAmountSelectionProps } from '../types';
+import './AmountSelections.scss';
+
+const SellAmountSelection = ({ trading }: SellAmountSelectionProps) => {
+  return (
+    <div className="AmountSelections">
+      <div className="amount-input-with-currency-symbol">
+        <CWTextInput
```

Comment:
shouldn't we make it type number?

Reply:
originally it was, those float values were used in calculations and needed to be displayed differently. So I settled with string storage and float parsing those values